funnel/models/rsvp.py

```diff
@@ -27,9 +28,9 @@ class Rsvp(TimestampMixin, db.Model):
     user_id = db.Column(None, db.ForeignKey('user.id'), nullable=False, primary_key=True)
     user = db.relationship(User)
 
-    status = db.Column(db.CHAR(1),
-        db.CheckConstraint('status IN (%s)' % ', '.join(["'%s'" % value for value in RSVP_STATUS.keys()])),
+    _state = db.Column('status', db.CHAR(1), StateManager.check_constraint('status', RSVP_STATUS),
```

Comment:
Please confirm the CHECK constraint hasn't changed. Migration required if it has.

Reply:
`db migrate` doesn't detect anything.